plugin/src/Theme.lua

```diff
@@ -11,10 +11,34 @@ local Theme = {
 	LightTextColor = Color3.fromRGB(160, 160, 160),
 }
 
-setmetatable(Theme, {
+local DarkTheme = {
+
+	-- Little unsure on how to change the color of the background on the plugin for dark mode...
+
+	ButtonFont = Enum.Font.GothamSemibold,
+	InputFont = Enum.Font.Code,
+	TitleFont = Enum.Font.GothamBold,
+	MainFont = Enum.Font.Gotham,
+	
+	AccentColor = Color3.fromRGB(89, 88, 88),
+	AccentLightColor = Color3.fromRGB(255, 146, 145),
+	PrimaryColor = Color3.fromRGB(53, 53, 53),
+	SecondaryColor = Color3.fromRGB(103, 103, 103),
+	LightTextColor = Color3.fromRGB(103, 103, 103)
+} 
+
+if Enum.UITheme == Enum.UITheme.Light then
```

Comment:
I think there's a different API for getting what the actual theme is, I imagine this will always return false?

Reply:
You'd be right. `settings().Studio.Theme`

Reply:
Just replaced that in the latest commit.